api/autocomplete.mjs

```diff
@@ -0,0 +1,21 @@
+import Postmates from "../services/Postmates.mjs";
+import { HTTPResponseError } from "../errors/http.mjs";
+
```

Comment:
is this the API that we will be using to autocomplete in the search bar?

Reply:
@dani5612 Yes, although that is not complete yet. Since we don't have a specific use case for that, I'll be adding that as well.